browser/extensions/api/settings_private/brave_prefs_util.cc

```diff
@@ -315,6 +317,18 @@ const PrefsUtil::TypedPrefMap& BravePrefsUtil::GetAllowlistedKeys() {
   // NFT pinning pref
   (*s_brave_allowlist)[kAutoPinEnabled] =
       settings_api::PrefType::PREF_TYPE_BOOLEAN;
+
+#if defined(TOOLKIT_VIEWS)
```

Comment:
Just double checking we need the `TOOLKIT_VIEWS` check here - this file is in extensions, so it might only exist on desktop :laughing:

Reply:
Yeah, I'm not sure about it. @simonhong , what do you think about this? `kSidebarShowOption` is guarded with TOOLKIT_VIEWS too, though it seems to be replacement of ENABLE_SIDEBAR.

Reply:
Hmm.. I picked `TOOLKIT_VIEWS` to mark sidebar is desktop UI feature.
Previously, `ENABLE_SIDEBAR` was used but realized that both are defined with same condition.
So, redundant `ENABLE_SIDEBAR` is removed and all things outside of views is replaced with `TOOLKIT_VIEWS`.

Actually, removing this will work but I want to use it here because this file `brave_prefs_util.cc` is not guarded by `toolkit_views` in the gn file. If this file is guarded by `toolkit_views` in gn, I think we don't need to use this flag.

Reply:
Yeah, unless we have `assert(tookit_views)` in `browser/extensions/BUILD.gn`, it'd be safer to have TOOLKIT_VIEWS in sources like this. FYI, the BUILD.gn also has conditional with toolkit_views.

```
  if (toolkit_views) {
    deps += [ "//brave/components/sidebar" ]
  }
```

So, we might want to check if assert(toolkit_views) is better, but I think we can deal with it in a separate issue.

Reply:
Filed an issue for it : https://github.com/brave/brave-browser/issues/29918